src/Deprecated70/Integer.extension.st

```diff
@@ -0,0 +1,19 @@
+Extension { #name : #Integer }
+
+{ #category : #'*Deprecated70' }
+Integer >> isMostLikelyPrime [
+	"See isProbablyPrimeWithK:andQ: for the algoritm description."
+	"Moving to Deprecated as it has no implementation"
+	| k q |
+	self <= 1 ifTrue: [^self error: 'operation undefined'].
+	self even ifTrue: [^self = 2].
+	k := 1.
+
+	q := self - 1 bitShift: -1.
+	[q odd] whileFalse: 
+			[q := q bitShift: -1.
+			k := k + 1].
+	
+	25 timesRepeat: [(self isProbablyPrimeWithK: k andQ: q) ifFalse: [^false]].
+	^true
+]
```

Comment:
Maybe it should have a deprecation warning too.

Reply:
Maybe we can even remove it directly, in the current image there is no implementation of the sent messages, so this isMostLikelyPrime produces a DNU. Also it is no used anywhere.

Reply:
If it produce a DNU then yes, maybe it's better to just remove it :)